src/main/java/com/algorand/algosdk/v2/client/Utils.java

```diff
@@ -1,19 +1,161 @@
 package com.algorand.algosdk.v2.client;
 
+import java.math.BigInteger;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.Set;
+
+import com.algorand.algosdk.crypto.Address;
+import com.algorand.algosdk.transaction.SignedTransaction;
+import com.algorand.algosdk.transaction.Transaction;
+import com.algorand.algosdk.transaction.Transaction.Type;
 import com.algorand.algosdk.v2.client.common.AlgodClient;
 import com.algorand.algosdk.v2.client.common.Response;
+import com.algorand.algosdk.v2.client.model.DryrunRequest;
+import com.algorand.algosdk.v2.client.model.Application;
+import com.algorand.algosdk.v2.client.model.ApplicationParams;
+import com.algorand.algosdk.v2.client.model.ApplicationStateSchema;
+import com.algorand.algosdk.v2.client.model.Account;
+import com.algorand.algosdk.v2.client.model.Asset;
 import com.algorand.algosdk.v2.client.model.NodeStatusResponse;
 import com.algorand.algosdk.v2.client.model.PendingTransactionResponse;
 
 public class Utils {
+
+    // https://github.com/algorand/go-algorand/blob/e466aa18d4d963868d6d15279b1c881977fa603f/libgoal/libgoal.go#L1089-L1090
+    private static Long defaultAppId = 1380011588L;
+
+    /**
+     * Construct a DryruynRequest object from a set of transactions
+     * 
+     * @param client           an Algod v2 client
+     * @param txns             the array of SignedTransactions that should be used
+     *                         to generate the DryrunRequest
+     * @param protocol_version The protocol version the dryrun should include
+     * @param latest_timestamp The latest timestamp the dryrun should include
+     * @param round            The agreement round or block height the dryrun should
```

Comment:
These parameters are for the other version of this function, could you make sure both functions are documented? I would write details in the version with 5 arguments and link to it from the version with 2 arguments, something like this:
> For additional details refer to {@link #createDryrun(AlgodClient, List<SignedTransaction>, String,
            Long, Long)}.

Reply:
nice, thanks!